.travis.yml

```diff
@@ -113,6 +113,11 @@ install:
     else
       echo "ZeroMQ directory already exists";
     fi
+  #############################################################################
+  # Conan
+  #############################################################################
+  - pip install --upgrade pip --user
+  - pip install conan --user
```

Comment:
You could add the `build-deps-from-source` and the `use conan` dependencies as two independent tests in the travis build matrix